edalize/vunit_hooks.py

```diff
@@ -1,30 +1,29 @@
-"""This module exports :class:`VUnitHooks` which can be used to implement advanced VUnit test cases"""
+"""This module exports :class:`VUnitHooks` which can be used to implement advanced VUnit test cases."""
 
 from vunit.ui import Library
 from vunit import VUnit
 from typing import Mapping, Collection
 
 
 class VUnitHooks(object):
-    """Derive the :class:`VUnitRunner` instance from this class and override its member functions if necessary"""
+    """Derive the :class:`VUnitRunner` instance from this class and override its member functions if necessary."""
 
     def __init__(self):
         pass
 
     def create(self) -> VUnit:
-        """Override this function to specify custom instantiation of VUnit"""
+        """Override this function to specify custom instantiation of VUnit."""
         return VUnit.from_argv()
 
     def handle_library(self, logical_name: str, vu_lib: Library):
-        """override this to customize each library, e.g. with additional simulator options"""
+        """Override this to customize each library, e.g. with additional simulator options."""
         pass
 
     def main(self, vu: VUnit):
-        """override this for final parametrization of the :class:`VUnit` instance, or for custom invokation of VUnit"""
+        """Override this for final parametrization of the :class:`~edalize.vunit.Vunit` instance, or for custom invocation of VUnit."""

```

Comment:
Actually, I'm not sure if this shouldn't instead be
```suggestion
        """Override this for final parametrization of the :class:`~vunit.ui.VUnit` instance, or for custom invocation of VUnit."""
```

Reply:
Yes @cmarqu, this override is intended to allow for custom configuration and invocation of the `vunit.ui.VUnit` instance.